XcodeServerSDK/XcodeServer.swift

```diff
@@ -106,7 +106,7 @@ public extension XcodeServer {
     }
     
     //API functionality
-    private func sendRequestWithMethod(method: HTTP.Method, endpoint: XcodeServerEndPoints.Endpoint, params: [String: String]?, query: [String: String]?, body: NSDictionary?, completion: HTTP.Completion) {
+    internal func sendRequestWithMethod(method: HTTP.Method, endpoint: XcodeServerEndPoints.Endpoint, params: [String: String]?, query: [String: String]?, body: NSDictionary?, completion: HTTP.Completion) {
```

Comment:
Maybe let's rely on `internal` being implicit and just not clarify any modifier when we want to use it?